dump_offload_main.cpp

```diff
@@ -15,8 +16,16 @@ int main()
     {
         auto bus = sdbusplus::bus::new_default();
         auto event = sdeventplus::Event::get_default();
-        openpower::dump::DumpOffloadManager manager(bus);
-        manager.offload();
+        // Changing a system from hmc-managed to non-hmc manged is a disruptive
+        // process (Power off the system, do some clean ups and IPL).
+        // Changing a system from non-hmc managed to hmc-manged can be done at
```

Comment:
So, is this app alive but, won't perform any action? 

OR Do we stop this app if the system changed with HMC support?

Reply:
we don't need to stop this app since, need to move more power vm communications like host dump delete/offload, etc to this app.